.pre-commit-config.yaml

```
@@ -126,7 +126,7 @@ repos:
         entry: python scripts/validate_unwanted_patterns.py --validation-type="bare_pytest_raises"
         types: [python]
         files: ^pandas/tests/
-        exclude: ^pandas/tests/(computation|extension|io)/
+        exclude: ^pandas/tests/extension
```

```suggestion
        exclude: ^pandas/tests/extension/
```
(just to make it clear it's a directory)